docs/workflow-executors.md

```diff
@@ -4,16 +4,55 @@ A workflow executor is a process that conforms to a specific interface that allo
 
 The executor to be used in your workflows can be changed in [the configmap](./workflow-controller-configmap.yaml) under the `containerRuntimeExecutor` key.
 
+## Emissary (emissary)
+
+> v3.1 and after
+
+**default in >= v3.3**
+
+This is the most fully featured executor.
+
+* Reliability:
+  * Works on GKE Autopilot
```

Comment:
Perhaps worth another discussion but do we want to include vendor-specific information in the docs?